apiserver/facades/client/cloud/cloud.go

```diff
@@ -1030,7 +1019,14 @@ func (api *CloudAPI) AddCloud(cloudArgs params.AddCloudArgs) error {
 			logger.Infof("force adding cloud %q of type %q to controller bootstrapped on cloud type %q", cloudArgs.Name, cloudArgs.Cloud.Type, controllerCloud.Type)
 		}
 	}
-	err = api.backend.AddCloud(common.CloudFromParams(cloudArgs.Name, cloudArgs.Cloud), api.apiUser.Name())
+
+	aCloud := common.CloudFromParams(cloudArgs.Name, cloudArgs.Cloud)
+	// All clouds must have at least one 'default' region, lp#1819409.
+	if len(aCloud.Regions) == 0 {
+		aCloud.Regions = []cloud.Region{{Name: "default"}}
```

Comment:
Can we define a DefaultCloudRegion const in juju/cloud since "default" is used her and in the CLI